src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -71,6 +72,12 @@ private boolean issueHasNewComments(TurboIssue issue, boolean hasMetadata) {
         }
     }
 
+    private static boolean containsUpdatedQualifier(List<String> filterQualifierNames){
```

Comment:
This method and the fields above could go in `Qualifier`. The name is a bit too generic for what it does also.